app/helpers/date_time_helper.rb

```diff
@@ -0,0 +1,13 @@
+module DateTimeHelper
+  # Show date in local timezone. Expected to be used
+  # by frontend JS library like date-fns or moment.js
+  def local_date(datetime)
+    tag.span(
```

Comment:
instead of being a span it should be a `<time>` element. See https://developer.mozilla.org/en-US/docs/Web/HTML/Element/time and https://github.com/thepracticaldev/dev.to/blob/5094f6be9c1d6065b233654def953dbf3dad650e/app/views/articles/show.html.erb#L155